.NET/Microsoft.Recognizers.Definitions.Common/Dutch/DateTimeDefinitions.cs

```diff
@@ -652,7 +653,7 @@ public static class DateTimeDefinitions
             { @"christmaseve", new string[] { @"kerstavond" } },
             { @"columbus", new string[] { @"columbusday" } },
             { @"thanksgiving", new string[] { @"thanksgivingday", @"thanksgiving", @"dankzeggingsdag" } },
-            { @"martinlutherking", new string[] { @"martinlutherkingday", @"martinlutherkingjrday" } },
+            { @"martinlutherking", new string[] { @"martinlutherkingday", @"martinlutherkingjrday", @"martinlutherkingdag", @"martin luther king dag", @"mlkdag" } },

```

Comment:
Values with spaces should not be needed in the dictionary as there is a simple normalization before querying, right?